smk/QC_2.smk

```diff
@@ -209,7 +209,7 @@ def taxonomy_plot_output():
     return(results)
 
 def smash_plot_output():
-    if omics == "metaG":
+    if omics == "metaG" and config['SOURMASH_max_abund'] > 0:
```

Comment:
Can you please switch to `validate_optional_key(config, 'SOURMASH_max_abund')` instead? We are moving away from direct access using `config[key]` so that variable types can be automatically verified. Please add an entry in `include/config_parser.smk`.